nav2_robot/src/ros_robot.cpp

```diff
@@ -23,17 +23,6 @@ namespace nav2_robot
 RosRobot::RosRobot(rclcpp::Node * node)
 : node_(node), initial_pose_received_(false), initial_odom_received_(false)
 {
-  // Open and parse the URDF file
```

Comment:
This has already been addressed in PR #175.

Reply:
Sure, that'll be handled when I rebase.